File: src/main/java/org/sindeaux/autoconsulta/entityes/Paciente.java
package org.sindeaux.autoconsulta.entityes;

import jakarta.persistence.Column;
import jakarta.persistence.Entity;
import jakarta.persistence.FetchType;
import jakarta.persistence.OneToMany;

import java.util.List;

@Entity
public class Paciente extends Pessoa{
    @Column(name = "NUM_CARTAO")
    private String numeroConvenio;

    @OneToMany(mappedBy = "paciente", fetch = FetchType.LAZY)
    private List<Consulta> consultas;
}

File: src/main/kotlin/com/sindeaux/autoconsulta/entity/Paciente.kt
package com.sindeaux.autoconsulta.entity

import Pessoa
import javax.persistence.Column
import javax.persistence.Entity
import javax.persistence.FetchType
import javax.persistence.OneToMany

@Entity
class Paciente : Pessoa() {

    @Column(name = "NUM_CARTAO")
    open var numCartao: String? = null

    @OneToMany(mappedBy = "paciente", fetch = FetchType.LAZY)
    open var consultas: List<Consulta>? = null;
}